### PR TITLE
Create pathways-of-buddhist-thought-four-essays_nyanamoli

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Pathways of Buddhist Thought Four Essays"
+authors:
+  - nyanamoli
+external_url: "https://www.bps.lk/olib/wh/wh052_Nyanamoli_Pathways-Of-Buddhist-Thought--Four-Essays.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10nsDXmod0VTuhqKjpj7fOKUrKc_-U9-e/view?usp=sharing"
+formats: [pdf]
+course: thought
```
Comment (buddhist-uni) [inline _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md:9]: I know I put it in thought (based, no doubt, on the title), but I'm now second guessing that... Perhaps (with its focus on faith and understanding) this booklet more properly belongs in right `view`... What do you think?

File: _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+tags:
+  - saddha
+  - cessation
```
Comment (buddhist-uni) [inline _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md:12]: I'm thinking `function` instead...

**Review by buddhist-uni — COMMENTED** [review]
Sorry for the delay... had to reskim the book as it's been a while since I read it the first time...

File: _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Pathways of Buddhist Thought: Four Essays"
+authors:
+  - nyanamoli
+external_url: "https://www.bps.lk/olib/wh/wh052_Nyanamoli_Pathways-Of-Buddhist-Thought--Four-Essays.html"
+drive_links:
+  - "https://drive.google.com/file/d/10nsDXmod0VTuhqKjpj7fOKUrKc_-U9-e/view?usp=sharing"
+  - "https://drive.google.com/file/d/1AVDLjlcrasmBvA01FLJjTFvt-yIxsTHh/view?usp=drive_link"
+formats: [pdf]
```
Comment (buddhist-uni) [inline _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md:9]: please remove this line (the drive links are now [pdf, epub] but that's the default value for formats anyway)

File: _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+course: view
```
Comment (buddhist-uni) [inline _content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md:10]: So you agree with `view`? I was genuinely asking 😅

**Comment by JJM317** [comment]
I can go either way regarding view or thought for this work. I think you
make a solid point that faith seems closely linked to a view, sammā or
asammā. Then again, sammā-saṅkappa also carries a connotation of resolve or
aspiration, not just thought (wholesome or unwholesome), so faith would
fall under that category. Actually, as I'm thinking about it more, we'd
leave a work on renunciation or ahimsa in thought or no? Renunciation falls
squarely into sammā-saṅkappa; ahimsa fits too, but also fits ethics, right?

But...the first and last essay do seem to primarily deal with ideas
relating to view. For instance, Bhante seems to be trying to lead the
reader to the Buddha's sammā-diṭṭhi over other views, and the last essay
deals with consciousness, nibbāna, etc.

It's an "amphibious' text, as it were. Seems there's just gonna be times
when a work or term can fit in multiple categories and we just have to
stick it somewhere. What are the plus/minuses of having it one over the
other? Is there a risk of a user missing out on this particular text? I
guess if they just do one course and not the other, then the answer is yes.
I'll defer to the head librarian :)

I removed [pdf] from the metadata. Hope I understood your comment
correctly. I was wondering which format (pdf or epub) to list since this
entry had two links and I was thinking of putting both. So, is it safe to
say if there are two Drive links, take format out?

Kindly,
~Joe

**Review by buddhist-uni — APPROVED** [review]
Okay, yeah I think view is better then. Thanks for helping me think it through. We'll discuss the "format" field in our meeting this week 😁

**Pull request merged**